10_rest/runnerdave/README.md

```diff
@@ -0,0 +1,92 @@
+# Puppy server
+
+Storage of Puppies with two different storage methods.
+
+## Pre-requisites
+
+You need to have [Go](https://golang.org/doc/install) installed
+
+## Build and install
+
+```
+go install ./...
```

Comment:
nit/personal preference, feel free to ignore: I prefer to just indent my code blocks in Markdown unless I specify the language with it, so in you case to render the same result that would be:

```markdown
## Build and install

    go install ./...
```

Reply:
much cleaner this way, thanks